services/src/main/java/org/exoplatform/officeonline/WOPIService.java

```diff
@@ -149,7 +163,9 @@ public WOPIService(SessionProviderService sessionProviders,
     PropertiesParam param = initParams.getPropertiesParam(TOKEN_CONFIGURATION_PROPERTIES);
     String secretKey = param.getProperty(SECRET_KEY);
     if (secretKey != null && !secretKey.trim().isEmpty()) {
-      activeCache.put(SECRET_KEY, secretKey);
+      byte[] decodedKey = Base64.getDecoder().decode(secretKey);
+      SecretKey key = new SecretKeySpec(decodedKey, 0, decodedKey.length, ALGORITHM);
+      activeCache.put(SECRET_KEY, key);
```

Comment:
Same issue with cache eviction as previously.

Reply:
Fixed in https://github.com/exo-addons/office-online/pull/2/commits/a0bd2084c19c9658b8aa5a48314c1653208a7876